src/screens/FeedScreen/RenderList.js

```diff
@@ -41,7 +44,9 @@ const RenderListFeed = (props) => {
     source = SOURCE_FEED_TAB,
     hideThreeDot = true,
     showAnonymousOption = false,
-    onHeaderOptionClicked = () => {}
+    onDeletePost,
+    isShowDelete,
+    isSelf
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New props have been added to the component. Ensure that these props are being passed correctly from the parent components and are used as intended.
```diff
- onHeaderOptionClicked = () => {}

+ onDeletePost,
+ isShowDelete,
+ isSelf
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->